src/Providers/FindologicServiceProvider.php

```diff
@@ -47,6 +47,22 @@ public function boot(
             return;
         }
 
+        if (!$configRepository->get(Plugin::CONFIG_SHOPKEY, false)) {
```

Comment:
@tomas-adeoweb 
See [comment](https://github.com/findologic/plugin-plentymarkets-ceres-api/pull/6/files#r228881098). We still have to check if the search was activated in the plugin specific configs.

Reply:
@ihanli The if with Plugin::CONFIG_ENABLED is checking this.